sql/core/src/test/scala/org/apache/spark/status/api/v1/sql/SqlResourceSuite.scala

```diff
@@ -203,3 +210,101 @@ class SqlResourceSuite extends SparkFunSuite with PrivateMethodTester {
   }
 
 }
+
+case class Person(id: Int, name: String, age: Int)
+case class Salary(personId: Int, salary: Double)
+
+/**
+ * Sql Resource Public API Unit Tests running query and extracting the metrics.
+ */
+class SqlResourceWithActualMetricsSuite extends SharedSparkSession with SQLMetricsTestUtils {
```

Comment:
nit: Shall we move this to a single file and leave SqlResourceSuite.scala unchanged?

Reply:
Moved to separate file